src/gas_saved.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+import click
+import pandas as pd
+from dotenv import load_dotenv
+from web3 import Web3
+
+from src.db.pg_client import pg_engine
+
+
+@click.command()
+@click.option(
+    "--batch_tx_hash",
+    help="The transaction hash of the batch you want to see the % gas saved.",
+    type=str
+)
```

Comment:
@gentrexha - what is this decorator for here?

Reply:
It's for the command line interface. Ref: https://click.palletsprojects.com/en/8.1.x/

Reply:
Cool stuff!